infra-examples/aws/eks/eks.tf

```diff
@@ -0,0 +1,178 @@
+locals {
+  cluster_autoscaler_tags = var.enable_cluster_autoscaler ? {
+    "k8s.io/cluster-autoscaler/${var.cluster_name}" = "owned"
+    "k8s.io/cluster-autoscaler/enabled"             = "true"
+  } : {}
+  post_bootstrap_user_data = var.post_bootstrap_user_data != null ? var.post_bootstrap_user_data : templatefile(
+    "${path.module}/templates/post_bootstrap_user_data.tpl",
+    {
+      registry_credentials = var.registry_credentials
+    }
+  )
+  # Define the default IAM role additional policies for all the node groups
+  # every element must define:
+  #   - A key for the policy. It can be any string
+  #   - A value which is the ARN of the policy to add
+  #   - An enable key which determines if the policy is added or not
+  node_group_defaults_iam_role_additional_policies = [
+  ]
+}
+
+data "aws_ami" "latest_ubuntu_eks" {
+  most_recent = true
+  owners      = ["099720109477"] # Canonical
+
+  filter {
+    name   = "name"
+    values = ["ubuntu-eks/k8s_${var.cluster_version}/images/hvm-ssd/ubuntu-${var.ubuntu_version}-amd64-server-*"]
+  }
+}
+
+module "eks" {
+  source                         = "terraform-aws-modules/eks/aws"
+  version                        = "~> 20.24"
+  cluster_name                   = var.cluster_name
+  cluster_version                = var.cluster_version
+  cluster_endpoint_public_access = true
+  vpc_id                         = module.vpc.vpc_id
+  subnet_ids                     = module.vpc.private_subnets
+  enable_irsa                    = true
+  cluster_tags                   = var.cluster_tags
+  tags                           = var.tags
+
+  create_cloudwatch_log_group = false
+  cluster_enabled_log_types   = []
+
+  cluster_addons = {
+    coredns = {
+      name = "coredns"
+    }
+    kube-proxy = {
+      name = "kube-proxy"
+    }
+    vpc-cni = {
+      name = "vpc-cni"
+    }
+    aws-ebs-csi-driver = {
+      name                     = "aws-ebs-csi-driver"
+      service_account_role_arn = module.ebs_csi_irsa_role.iam_role_arn
+    }
+  }
+
+  # The security group rules below should not conflict with the recommended rules defined
+  # here: https://github.com/terraform-aws-modules/terraform-aws-eks/blob/v19.21.0/node_groups.tf#L128
+  node_security_group_additional_rules = {
```

Comment:
I wonder if these rules are still required. I understand the SSH rule, but not sure about the other two. Could you please check what are the default rules defined in the module?

Reply:
@Henrrypg Yes, proceed with the deletion